### ISSUE TITLE
5.95.4 freeze on MBP (Mid 2014) when wlan network is used.

### ISSUE THREAD

**Issue by Megachip** [OpenELEC/OpenELEC.tv#4295]
See title. Connecting works fine, but when the first Network request is going out, openelec freezes.

**Comment by chewitt** [comment]
SSH in and run "dmesg | paste" and share the URL (not the log) here

**Comment by Megachip** [comment]
@chewitt 

Thats the part which wouldn't work, cause it freeze on first network activity. Didn't matter if this are incomping ssh/http or outgoing (e.g. updating repositories).

**Comment by chewitt** [comment]
how do you know (can you prove) it's on network activity? .. did the card associate to a network and get an IP address properly beforehand? - also which broadcom card do yo have?

**Comment by Megachip** [comment]
@chewitt correct. And when i disable wireless lan before openelec runs and runs and runs ;)
Activating wlan (and connect), getting ip and freeze on frist traffic.

Is there a way to do a persistant logging? or possible get an console before enable the wlan?

**Comment by chewitt** [comment]
It's possible to set netconsole in kernel boot params and stream the kernel log to another host running netcat ..although I never had much success with that myself. Are you using Generic or Nvidia_legacy? - let me know and I will create a test build with additional patches from the Ubuntu driver package.

**Comment by Megachip** [comment]
@chewitt generic. legacy will not run on the macbook. (afaik it is an 750M).

Hmm, no way to logging to a file on the storage partition?

**Comment by chewitt** [comment]
http://chewitt.openelec.tv/test/OpenELEC-Generic.x86_64-6.0-devel-20150831195234-r21241-g494f51d.tar <= update to that build and see if it improves things?

**Comment by Megachip** [comment]
Didn't detect/find the wireless card anymore. So sure, it wouln't freeze 
anymore on network traffic ;) Almost the same as diasble the network 
manually ;)

Another thing, the touchpad isn't recognised (same in beta 4).



Am 31.08.2015 um 18:43 schrieb Christian Hewitt:

**Comment by Megachip** [comment]
If it is relevant, it is an A1398 (EMC 2881). 

Network Adapter is listened as BCM43XX

**Comment by chewitt** [comment]
I want to know the exact broadcom chipset as shown in "lspci" .. 43XX is not helpful

**Comment by Megachip** [comment]
@chewitt sure, how? OS X has no lspci and on openelec I didn't know how to switch to terminal.

**Comment by chewitt** [comment]
You have a Broadcom BCM4360. Google reveals lots (and lots) of reports of the wl driver freezing and crashing with multiple card type after kernel v3.18+ e.g. https://bbs.archlinux.org/viewtopic.php?id=192287 ..but the solution is one of the patches included in that build. Any further diagnosis will need you to SSH into the box and give me some logs that show output from "journalctl --no-pager -b -0 -f" up to the point where it freezes - you might see something.

**Comment by Megachip** [comment]
I will do my best, BUT:
1. Beta 4 crashes immediately on the first network connection = > No SSH Possible.
2. Your "fixed" build did not recognized the network interface = > No SSH Possible.

So i still need a local terminal to provide anything :( So any possibility to get on tty? Or an xterm plugin?

**Comment by Megachip** [comment]
So, a 3rd try:

How to get an local terminal on openelec? CTRL+ALT+F* has no tty running :(

**Comment by MilhouseVH** [comment]
Add "textmode" (without quotes) to the end of your kernel command line -
OpenELEC will boot into a console.

**Comment by Megachip** [comment]
Cause I have no system which supports ext4 atm, I'll choose the autostart.sh way to pipe the log to storage.

Cause here not txt files are supported as attachment:
http://xbmclogs.com/p0wpumsgf
Will post the log of the 95.4 later after downgrade

**Comment by Megachip** [comment]
Bug reintreduce with 6.0.1
